deployment/ansible/raster-foundry.yml

```diff
@@ -15,6 +15,6 @@
     - name: Change directory on login
       lineinfile: dest=/home/vagrant/.bashrc regexp=^raster-foundry_PROFILE line="cd /opt/raster-foundry/"
 
-    - name: Set Environment variable for AWS profile
-      lineinfile: dest=/etc/environment regexp=^AWS_PROFILE line="AWS_PROFILE={{aws_profile}}"
```

Comment:
I'll switch it back, it wasn't broken before so no reason to mess things up by accident by switching it